### PR TITLE
Fixes for PHP 7.2

### PR DESCRIPTION
**Pull request by amyboyd**

### CODE REVIEW

File: src/dawguk/GarminConnect.php [hunk 1]
```diff
@@ -327,7 +327,11 @@ public function getUser()
      */
     public function getUsername()
     {
-        return $this->getUser()->username;
+        $user = $this->getUser();
```
Comment (dawguk) [inline src/dawguk/GarminConnect.php]:
```suggestion
        $objUser = $this->getUser();
```

File: src/dawguk/GarminConnect.php [hunk 2]
```diff
@@ -327,7 +327,11 @@ public function getUser()
...
+        if (!$user) {
```
Comment (dawguk) [inline src/dawguk/GarminConnect.php]:
```suggestion
        if (!$objUser) {
```

File: src/dawguk/GarminConnect.php [hunk 3]
```diff
@@ -327,7 +327,11 @@ public function getUser()
...
+            return null;
+        }
+        return $user->username;
```
Comment (dawguk) [inline src/dawguk/GarminConnect.php]:
```suggestion
        return $objUser->username;
```

**Comment by amyboyd** [comment]
@dawguk Updated!

**Comment by dawguk** [comment]
Awesome, thanks!

**Pull request merged**